shell/platform/windows/flutter_window_winuwp.cc

```diff
@@ -0,0 +1,337 @@
+}
+
+void FlutterWindowWinUWP::OnPointerPressed(
+    winrt::Windows::Foundation::IInspectable const&,
+    winrt::Windows::UI::Core::PointerEventArgs const& args) {
+  auto x = GetPosX(args);
+  auto y = GetPosY(args);
+
+  binding_handler_delegate_->OnPointerDown(
+      x, y, FlutterPointerMouseButtons::kFlutterPointerButtonMousePrimary);
+}
+
+void FlutterWindowWinUWP::OnPointerReleased(
+    winrt::Windows::Foundation::IInspectable const&,
+    winrt::Windows::UI::Core::PointerEventArgs const& args) {
+  auto x = GetPosX(args);
+  auto y = GetPosY(args);
+
+  binding_handler_delegate_->OnPointerUp(
+      x, y, FlutterPointerMouseButtons::kFlutterPointerButtonMousePrimary);
+}
+
+void FlutterWindowWinUWP::OnPointerMoved(
+    winrt::Windows::Foundation::IInspectable const&,
+    winrt::Windows::UI::Core::PointerEventArgs const& args) {
+  auto x = GetPosX(args);
+  auto y = GetPosY(args);
+
+  binding_handler_delegate_->OnPointerMove(x, y);
+}
+
+void FlutterWindowWinUWP::OnPointerWheelChanged(
+    winrt::Windows::Foundation::IInspectable const&,
+    winrt::Windows::UI::Core::PointerEventArgs const& args) {
+  auto x = GetPosX(args);
+  auto y = GetPosY(args);
+  auto delta = args.CurrentPoint().Properties().MouseWheelDelta();
+  binding_handler_delegate_->OnScroll(x, y, 0, -delta, 1);
+}
+
+double FlutterWindowWinUWP::GetPosX(
+    winrt::Windows::UI::Core::PointerEventArgs const& args) {
+  const double inverse_dpi_scale = GetDpiScale();
+
+  return (args.CurrentPoint().Position().X - xbox_overscan_x_offset_) *
+         inverse_dpi_scale;
+}
+
+double FlutterWindowWinUWP::GetPosY(
+    winrt::Windows::UI::Core::PointerEventArgs const& args) {
+  const double inverse_dpi_scale = GetDpiScale();
+  return static_cast<double>(
+      (args.CurrentPoint().Position().Y - xbox_overscan_y_offset_) *
+      inverse_dpi_scale);
+}
+
+void FlutterWindowWinUWP::OnBoundsChanged(
+    winrt::Windows::UI::ViewManagement::ApplicationView const& app_view,
+    winrt::Windows::Foundation::IInspectable const&) {
+#ifndef USECOREWINDOW
+  if (binding_handler_delegate_) {
+    auto bounds = GetBounds(current_display_info_, true);
+
+    binding_handler_delegate_->OnWindowSizeChanged(
+        static_cast<size_t>(bounds.width), static_cast<size_t>(bounds.height));
+    render_target_.Size({bounds.width, bounds.height});
+  }
+#endif
+}
+
+void FlutterWindowWinUWP::OnKeyUp(
+    winrt::Windows::Foundation::IInspectable const&,
+    winrt::Windows::UI::Core::KeyEventArgs const& args) {
+  // TODO(clarkezone) complete keyboard handling including
+  // system key (back), unicode handling, shortcut support,
+  // handling defered delivery
+  // https://github.com/flutter/flutter/issues/70202
+  auto status = args.KeyStatus();
+  unsigned int scancode = status.ScanCode;
+  int key = static_cast<int>(args.VirtualKey());
+  char32_t chararacter = static_cast<char32_t>(key | 32);
+  int action = 0x0101;
```

Comment:
Let's leave a TODO to refactor the delegate's `OnKey` so we're not hard-coding magic Win32 values into the UWP code.